Demo/src/main/java/com/paypal/android/ui/card/CardFragment.kt

```diff
@@ -119,31 +119,43 @@ class CardFragment : Fragment() {
         findNavController().navigate(R.id.action_cardFragment_to_testCardFragment)
     }
 
+    private fun updateStatusTextWithCardResult(result: CardResult, orderStatus: String?) {
```

Comment:
We'll want to support AUTHORIZE / CAPTURE for all payment types, not just Card. Maybe let's move this toggle to the settings, where the environment toggle lives so it can apply to all payment method demos?

Reply:
Yeah good callout, we only did Card here. Are y'all feeling the UX of the settings menu though? To me, putting Capture / Authorize as a Drop Down option made more sense here. I feel like it makes our SDK capabilities more discoverable for people who've never used the Demo app.

Reply:
I don't feel too strongly about where the toggle goes. But we def wanna support AUTH/CAPTURE for the other 2 payment methods.